index.html

```diff
@@ -291,6 +291,31 @@ <h2>About me</h2>
       </div>
     </main>
 
+    <footer>
+      <div class="form-container">
+        <form>
+          <h1 class="form-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+          <div class="form-group">
+            <div class="form-control">
+              <label>
+                  <input type="text" name="full_name" placeholder="Full name">
+              </label>
+            </div>
+
+            <div class="form-control">
+              <label>
+                  <input type="text" name="email" placeholder="Email address">
+              </label>
+            </div>
+            <div class="form-control">
+              <textarea name="message" cols="30" maxlength="500" rows="10"></textarea>
+            </div>
+            <button type="submit" class="form-button">Get in touch</button>
+          </div>
+        </form>
+      </div>
+    </footer>
```

Comment:
Kindly include the social media icons at the bottom of the page like in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Figma Design:
![image](https://user-images.githubusercontent.com/38088818/152543193-4feea119-e95b-40f7-bf97-4120a1671745.png)

Your Design:
![image](https://user-images.githubusercontent.com/38088818/152542943-ef2f2679-1c81-45c3-a3e8-5a952c3f38f1.png)